### ISSUE TITLE
defaultBadgeData is referred as a function in Tutorial but is an object 

### ISSUE THREAD

**Issue by pr7prashant** [badges/shields#9501]
I was going through the documentation for creating a new badge and found a mismatch.

Here, `defaultBadgeData` is defined as an object.
https://github.com/badges/shields/blob/531d2acf5cf899ce26be459300845e2fef5c4cab/doc/TUTORIAL.md?plain=1#L190

In the description, `defaultBadgeData()` is mentioned as a callable.
https://github.com/badges/shields/blob/531d2acf5cf899ce26be459300845e2fef5c4cab/doc/TUTORIAL.md?plain=1#L226

**Expected Behaviour :**
`defaultBadgeData()` should be `defaultBadgeData` in the description.

**Comment by chris48s** [comment]
This is probably a hangover that should have been updated a long time ago.

Static class properties were introduced in node 12. Before that, everything had to be static functions that returned a value, so there was a point in the past where this was a function.

After we upgraded to node 12, we did a whole load of PRs like this https://github.com/badges/shields/pull/6566/files converting all the functions to values.

This reference to `defaultBadgeData` being a function in the documentation is probably from before we did that.

Up for submitting a PR to fix it?
